### PR TITLE
Adding Yaml file

### PR DESCRIPTION
**Pull request by smendez93**
This took me way longer than I expected it to.

### CODE REVIEW

File: people/2017/spring/SharM.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#hfoss
+name: Sharlene Mendez
+avatar: http://1.gravatar.com/avatar/a03174311293fc0f109cd5095917d70a
+blog: https://sharlenemsite.wordpress.com/
+feed:
```
Comment (jflory7) [inline people/2017/spring/SharM.yml]: The feed of your blog is `https://sharlenemsite.wordpress.com/feed/`, which is an RSS feed. The [HFOSS Planet](https://people.rit.edu/djaigm/planet/hfoss/) needs an RSS feed to put your blog posts there. Although, if you are using categories for your blog, you may want to use an appropriate category for HFOSS posts so all of your blog posts don't wind up there (e.g. https://sharlenemsite.wordpress.com/category/foss/feed/).

File: people/2017/spring/SharM.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#hfoss
+name: Sharlene Mendez
+avatar: http://1.gravatar.com/avatar/a03174311293fc0f109cd5095917d70a
+blog: https://sharlenemsite.wordpress.com/
+feed: 
+email: snm1546@rit.edu
+major: Game Design and Development
+irc: SharM
+rit_dce: None
```
Comment (jflory7) [inline people/2017/spring/SharM.yml]: For reference, your `rit_dce` is your RIT computer account / username (i.e. `snm1546`). Whether you want to add this to the file is optional.

File: people/2017/spring/SharM.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+forges:
+  - https://github.com/smendez93
+
+bio:
```
Comment (jflory7) [inline people/2017/spring/SharM.yml]: It was either this project or the [FOSSProfiles](https://github.com/FOSSRIT/FOSSProfiles/) repo, but I think it requires there to be some kind of content for the YAML field. Even if it's a few characters, I think OfCourse (i.e. the course software) requires this to build successfully.

**Comment by ritjoe** [comment]
The file name needs to end in the extension ".yaml" with an "a" and not ".yml" as in this commit.

**Pull request closed without merging**